AWSAppSyncUnitTests/AppSyncSubscriptionWithSyncTests.swift

```diff
@@ -0,0 +1,203 @@
+                                                                                                                                                deltaQueryHandler: emptyDeltaQueryHandler,
+                                                                                                                                                subscriptionResultHandler: emptySubscriptionResultHandler,
+                                                                                                                                                subscriptionMetadataCache: nil,
+                                                                                                                                                syncConfiguration: syncConfiguration,
+                                                                                                                                                handlerQueue: queue)
+        
+        let subscriptionWithSync2 = AppSyncSubscriptionWithSync<AWSAppSyncClient.EmptySubscription, GraphGetQuery, AWSAppSyncClient.EmptyQuery>(appSyncClient: appsyncClient!,
+                                                                                                                                                baseQuery: getQuery,
+                                                                                                                                                deltaQuery: emptyQuery,
+                                                                                                                                                subscription: emptySubscription,
+                                                                                                                                                baseQueryHandler: getQueryHandler,
+                                                                                                                                                deltaQueryHandler: emptyDeltaQueryHandler,
+                                                                                                                                                subscriptionResultHandler: emptySubscriptionResultHandler,
+                                                                                                                                                subscriptionMetadataCache: nil,
+                                                                                                                                                syncConfiguration: syncConfiguration,
+                                                                                                                                                handlerQueue: queue)
+        let result1 = subscriptionWithSync1.getOperationHash()
+        let result2 = subscriptionWithSync2.getOperationHash()
+        XCTAssertEqual(result1, result2, "Hash value should be equal for the two sync operation")
+        
+    }
+    
```

Comment:
Please add a test that ensures that cases that specify the same operation for both `baseQuery` and `deltaQuery` work as expected.

Reply:
Added